data_structures/linked_list/ascendingPriorityQueue.c

```diff
@@ -0,0 +1,271 @@
+/* Ascending priority queue using Linked List - Program to implement Ascending priority queue using Linked List */
+
+#include  <stdio.h>
+#include  <stdlib.h>  
+#define  NULL  0
```

Comment:
this is a C++ syntax.
In C, NULL can be defined as:
```C
#define NULL ((void*)0)
```
Or it can be avoided altogether

Reply:
Done.